webapp/sources/rudder/rudder-web/src/main/elm/sources/GroupCompliance/ViewNodesCompliance.elm

```diff
@@ -0,0 +1,90 @@
+module GroupCompliance.ViewNodesCompliance exposing (..)
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onClick, onInput)
+import List
+import List.Extra
+import String
+import Tuple3
+import Dict
+
+import GroupCompliance.ApiCalls exposing (..)
+import GroupCompliance.DataTypes exposing (..)
+import GroupCompliance.ViewUtils exposing (..)
+import Compliance.Utils exposing (displayComplianceFilters, filterDetailsByCompliance)
+
+
+displayNodesComplianceTable : Model -> Html Msg
+displayNodesComplianceTable model =
+  let
+    filters = model.ui.nodeFilters
+    complianceFilters = model.ui.complianceFilters
+    fun     = byNodeCompliance model complianceFilters
+    col     = "Node"
+    childs  = case model.groupCompliance of
+      Just dc -> dc.nodes
+      Nothing -> []
+    childrenSort = childs
+      |> List.filter (\n -> (filterSearch filters.filter (searchFieldNodeCompliance n)))
+      |> List.filter (filterDetailsByCompliance complianceFilters)
+      |> List.sortWith sort
+
+    (children, order, newOrder) = case sortOrder of
+       Asc -> (childrenSort, "asc", Desc)
+       Desc -> (List.reverse childrenSort, "desc", Asc)
+
+    rowId = "by" ++ col ++ "s/"
+    rows = List.map Tuple3.first fun.rows
+    (sortId, sortOrder) = Dict.get rowId filters.openedRows |> Maybe.withDefault (col, Asc)
+    sort =   case List.Extra.find (Tuple3.first >> (==) sortId) fun.rows of
+      Just (_,_,sortFun) -> (\i1 i2 -> sortFun (fun.data model i1) (fun.data model i2))
+      Nothing -> (\_ _ -> EQ)
+    isGlobalMode = isGlobalCompliance model
+  in
+    ( if model.ui.loading then
+      generateLoadingTable
+      else
+      div[]
+      [ div [class "table-header extra-filters"]
+        [ div [class "d-inline-flex align-items-baseline pb-3 w-25"]
+          [
+            div [class "btn-group yesno"]
+            [ label [class ("btn btn-default" ++ if isGlobalMode then " active" else ""), style "box-shadow" (if isGlobalMode then "inset 0 3px 5px rgba(0,0,0,.125)" else "none"), onClick (LoadCompliance GlobalCompliance)]
```

Comment:
The styling is done "by-hand" here for testing, it is currently broken since the [bootstrap 5 migration](https://github.com/Normation/rudder/pull/5169), the shadow is no longer present when the button is active...

(This should be fixed globally in another PR)